.github/workflows/tests_skip.yaml

```diff
@@ -20,16 +20,37 @@ jobs:
     steps:
       - run: 'echo "skipping"'
 
-  run_browser_tests_aws:
-    strategy:
-      matrix:
-        shard_number: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
+  setup-browser-test-matrix:
     runs-on: ubuntu-latest
+
+    outputs:
+      matrix: ${{ steps.build-matrix.outputs.matrix }}
+
     steps:
-      - run: 'echo "skipping"'
+      - id: build-matrix
+        env:
+          # Need to access the branch name differently between 
+          BRANCH_NAME: ${{ github.event_name == 'pull_request' && github.head_ref || github.ref_name }}
+        run: |
+          # Temporarily ignore errors so we can check grep for matches
```

Comment:
this looks like it might be getting read as a part of the script and not a comment?

Reply:
Yep it is being interpreted as part of the script. That's on purpose.. I want to maintain locality with the specific `set +e/-e` commands since that can change the behavior of the step if not set correctly.